model/reactions.tsv

```diff
@@ -13072,3 +13072,5 @@ rxns	rxnKEGGID	rxnBiGGID	rxnEHMNID	rxnHepatoNET1ID	rxnREACTOMEID	rxnRecon3DID	rx
 "MAR10131"	""	""	""	""	""	""	""	""	"RCR90175"	""	0	""	""	"HMR_10131"
 "MAR10132"	""	""	""	""	""	""	""	""	"RCR90179"	""	0	""	""	"HMR_10132"
 "MAR13083"	""	""	""	""	""	""	""	""	""	""	0	""	""	"HMR_10133"
+"MAR13084"	""	""	""	""	""	""	""	""	""	""	0	""	""	""
+"MAR13085"	""	""	""	""	""	""	""	""	""	""	0	""	""	""
```

Comment:
It would be great to have some cross-references here, if possible.

Reply:
I could not find

Reply:
Alright, thanks for giving it a shot.

Reply:
Maybe check MNX, e.g. https://www.metanetx.org/equa_info/MNXR97136

Reply:
Ah, but maybe that's specific to transport to/from extracellular...

Reply:
> Ah, but maybe that's specific to transport to/from extracellular...

@JonathanRob It seems to be the case